dashboard/db/migrate/20160519000515_drop_plc_enrollment_task_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class DropPlcEnrollmentTaskAssignments < ActiveRecord::Migration
+  def change
+    drop_table :plc_enrollment_task_assignments
```

Comment:
Yeah - I've been pondering what to do about this.

I can make it reversible by creating up / down migrations and recreating the table. But in truth, I don't want anyone ever reversing this migration - you can't uncross this rubicon.

If we don't revert migrations except in emergency situations, then I'm fine putting in an up/down to recreate the table.